generated/nifake/unit_tests/test_session.py

```diff
@@ -151,8 +157,25 @@ def test_session_context_manager_close_with_error(self):
             assert e.code == test_error_code
             assert e.description == test_error_desc
 
-    # Methods
+    # Session locking
+    def test_lock_session_none(self):
+        with nifake.Session('dev1') as session:
+            session.lock()
+            self.patched_library.niFake_LockSession.assert_called_once_with(_matchers.ViSessionMatcher(SESSION_NUM_FOR_TEST), None)
+
+    def test_unlock_session_none(self):
+        with nifake.Session('dev1') as session:
+            session.unlock()
+            self.patched_library.niFake_UnlockSession.assert_called_once_with(_matchers.ViSessionMatcher(SESSION_NUM_FOR_TEST), None)
 
+    def test_lock_context_manager(self):
```

Comment:
consider adding test in which you exit `with` block by raising.

Reply:
Added